card/mod/01_core/set/all/templating.rb

```diff
@@ -12,29 +12,34 @@ def template
   # note that a *default template is never returned for an existing card.
   @template ||= begin
     @virtual = false
+
+    # NEW CARDS
     if new_card?
-      default_card = rule_card :default, :skip_modules=>true
+      default = rule_card :default, :skip_modules=>true
 
       dup_card = dup
-      dup_card.type_id = default_card ? default_card.type_id : Card.default_type_id
-
+      dup_card.type_id = default ? default.type_id : Card.default_type_id
 
-      if content_card = dup_card.structure_rule_card
+      if structure = dup_card.structure_rule_card
         @virtual = true if junction?
-        content_card
+        self.type_id = structure.type_id if assign_type_to?(structure)
```

Comment:
no, it's a virtual card thing.  In general, you're not allowed to set the type of a virtual card to be something other than what the structure rule defines.  We had a weird case where the url (or nest or something) was trying to override that type.

That's prevented in validation for real cards.

True that it shouldn't really be in this pr. my bad.